WcaOnRails/app/controllers/translations_controller.rb

```diff
@@ -49,36 +49,38 @@ def update
     @pr_url = Octokit.create_pull_request(upstream_repo, "master", "#{user_login}:#{branch_name}", message, pr_description_for(current_user, locale))[:html_url]
   end
 
+  # rubocop:disable Style/NumericLiterals
```

Comment:
Out of curiosity, what does rubocop *want* you to do here instead?

Reply:
He would like us to use insert `_` for long numbers, e.g. `32_229` instead of `32229`. While it looks neat in general, I don't think it makes sense in this case. (Especially that from now on, we'll always copy the id from unverified PR description and paste it in the code). Does that make sense?

Reply:
Ah yeah, I totally agree with your decision here.